tools/devtool

```diff
@@ -1082,69 +1211,133 @@ cmd_prepare_release() {
     rm -f "$FC_ROOT_DIR/CHANGELOG.md="
 }
 
-# Create a tag for the specified release.
-# The tag text will be composed from the changelog contents enclosed between the
-# specified release number and the previous one.
-# Args:
-#   $1  release number.
-# Example: `devtool tag 0.42.0`
+# Publish a Firecracker draft release to GitHub.
+# For this one needs to pass a GitHub API access token.
+# You can find instructions on how to create one: https://github.blog/2013-05-16-personal-api-tokens/.
+# If the tag associated with the intended release already exists or if the user has not specified
+# a GitHub API token, the script exit.
+# Example: `devtool release -v 0.42.0 -b main -t token`
 #
-cmd_tag() {
+cmd_release() {
+    local prerelease="false"
+    local repo="firecracker"
+    local remote="origin"
 
     # Parse any command line args.
     while [ $# -gt 0 ]; do
         case "$1" in
-            "-h"|"--help")      { cmd_help; exit 1;    } ;;
-            *)                  { version="$1"; break; } ;;
+            "-h"|"--help")      { help_release; exit 1; } ;;
+            "-v"|"--version")
+                shift
+                local version="$1"
+                ;;
+            "-t"|"--token")
+                shift
+                local token="$1"
+                ;;
+             "-u"|"--username")
+                shift
+                local user="$1"
+                ;;
+              "-r"|"--repo")
+                shift
+                repo="$1"
+                ;;
+              "-m"|"--remote")
+                shift
+                remote="$1"
+                ;;
+              "-p"|"--prerelease")
+                shift
+                prerelease="true"
+                ;;
+            *)
```

Comment:
Nit: it might make it easier to navigate through the code if we order these alphabetically.

Reply:
My intention was to place them in an order that reflects the importance of the parameters (version, token and user) are mandatory). I d like to leave them like this for this reason

Reply:
I agree, let's keep them as they are.